src/custom/ecospheres/views/HomeView.vue

```diff
@@ -32,29 +33,12 @@ useHead({
 <template>
   <div class="datagouv-components">
     <section class="fr-container fr-py-12v">
-      <div class="fr-grid-row fr-grid-row--gutters">
-        <div class="fr-col">
-          <h1>{{ homepageTitle }}</h1>
-          <p>
-            <i>ecologie</i><strong>.data.gouv</strong><i>.fr</i> référence et
-            centralise les données de la transition écologique.
-          </p>
-          <p>
-            Le catalogue rassemble les données utiles au déploiement des
-            politiques publiques portées par le Ministère de la Transition
-            écologique et de la Cohésion des territoires.
-          </p>
-        </div>
-        <div class="fr-col-md-6 fr-hidden fr-unhidden-md fr-grid-row--center">
-          <img
-            src="../assets/home-global-schema.svg"
-            alt=""
-            class="illustration"
-            width="441"
-            height="290"
-          />
-        </div>
-      </div>
+      <!-- eslint-disable-next-line vue/no-v-html -->
+      <h1 class="main-title-v2" v-html="homepageTitle"></h1>
+      <SearchComponent
+        search-label="Recherche"
+        placeholder="Aménagement des villes dans les jeux de données"
```

Comment:
Le placeholder est difficile à comprendre. Il mélange le query et le filtre en une seule phrase. D'autant plus compliqué à comprendre avant de savoir que la search bar permet de rechercher plusieurs types d'objets.

Reply:
As-tu des suggestions de placeholder ?

Reply:
1 - Avez-vous une suggestion de placeholder svp ? @narduin, @DanFluture275 & @thomas-duport (que je ne trouve pas dans github)
2 - @narduin mon implémentation d'un placeholder facultatif est à revoir ?

Reply:
Il faudrait à minima ajouter un label au-dessus du multiselect dont le `for` reprend l'`id` de ce dernier. [comme ici](https://github.com/opendatateam/udata-front-kit/blob/377dc21a21522384b12535fad96e1ec3e014d07b/src/components/header/HeaderSearch.vue#L61-L64)

Pour la valeur du placeholder, utiliser un exemple de bouquet ou de dataset (ou une combinaison des deux mais moins clair) :

- Itinéraires fraicheur
- Diagnostics de performance énergétique
- Horaires des TGV
- Itinéraires fraicheur/Horaires des TGV

À mon avis il faudrait expliciter ce qu'il est possible de faire avec ce champ de recherche car le placeholder ne peut pas représenter tous les cas d'usage. Ça voudrait dire ajouter une description comme on peut avoir sur la page de création de bouquet.

![image](https://github.com/user-attachments/assets/7416e750-ae4b-46ff-8f7b-2cec5eaeb86f)

Reply:
Remarque pertinente, vu qu'on s'éloigne de la maquette est-il possible d'avoir confirmation de @DanFluture275 svp ?